packages/protocol/contracts-0.8/governance/Validators.sol

```diff
@@ -890,7 +896,7 @@ contract Validators is
 
   /**
    * @notice Computes epoch payments to the account
-   * @param account The validator signer of the validator to distribute the epoch payment to.
+   * @param account The validator account of the validator to distribute the epoch payment to.
```

Comment:
IIUC, this is just fixing the natspec that was wrong rather than this changing behaviour, right?

Reply:
correct, as this function was already expecting the account.